spec/rspec/rails/example/rails_example_group_spec.rb

```diff
@@ -1,6 +1,10 @@
 module RSpec::Rails
   RSpec.describe RailsExampleGroup do
     if ::Rails::VERSION::MAJOR >= 7
+      class CurrentSample < ActiveSupport::CurrentAttributes
```

Comment:
What fails exactly without this?
At a glance it may looks that it’s the following example that check for tagged logger, but I guess it’s not, is it?

Reply:
Its just setup...